### PR TITLE
Multivarainte fit

### LINKED ISSUE #15676: ENH: Implement `multivariate_normal.fit`
### Is your feature request related to a problem? Please describe.

`multivariate_normal.fit` is not available, but is trivial to implement.

### Describe the solution you'd like.

Should be available.

### Describe alternatives you've considered.

Doing it yourself. It is inconvenient.

### Additional context (e.g. screenshots, GIFs)

```python
def fitGaussian(points):
    m = np.mean(points, axis=0)
    ms = points - m
    c = ms.T @ ms / (len(points) - 1)
    return scipy.stats.multivariate_normal(mean=m, cov=c)
```

### PR DESCRIPTION
**Pull request by IlayMenahem**
ENH: Implement multivariate_normal.fit.

Implement multivariate_normal.fit, MLE was the method of choice
for the Implemention also "Closes #15676".

### CODE REVIEW

**Comment by IlayMenahem** [comment]
last test had sample size which way too small, i've increased it's size from 500 to 1000 which on my machine fixed the assertion error

File: scipy/stats/_multivariate.py [hunk 1]
```diff
@@ -760,6 +760,36 @@ def rvs(self, mean=None, cov=1, size=1, random_state=None):
...
+        Notes
+        -----
+        This function uses MLE which probably 
+        isn't the best way to fit multivariante normal disterbution
+
+        References
+        ----------
+        https://numpy.org/doc/stable/reference/generated/numpy.cov.html?highlight=cov#numpy.cov
+
+        .. versionadded:: 1.9.4
```
Comment (j-bowhay) [inline scipy/stats/_multivariate.py:785]: Not in a position to review this but this won't go in until 1.10.0 at the earliest.
```suggestion
        .. versionadded:: 1.10.0
```

**Comment by IlayMenahem** [comment]
i'll have to look into why this error occurred
'isfinite' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''

**Comment by IlayMenahem** [comment]
the problem boils down to in ability to access cov of a multivariate_normal_frozen during test, do you know how could it be done?

the error given to when i tested the method
AttributeError: 'multivariate_normal_frozen' object has no attribute 'cov'

**Pull request closed without merging**